route53_crossfeed_app.tf

```diff
@@ -300,13 +310,23 @@ resource "aws_route53_record" "crossfeed_staging_api_CNAME1" {
 resource "aws_route53_record" "crossfeed_staging_api_CNAME2" {
   provider = aws.route53resourcechange
 
-  name    = "_95c4bc30d3e15e6b58dc96250e3661a4.api.staging.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  name    = "staging-cd.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
   records = ["_73aacddb56ac20ef77492c962bf006d8.jfrzftwwjs.acm-validations.aws"]
   ttl     = 300
   type    = "CNAME"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "crossfeed_staging_api_CNAME3" {
+  provider = aws.route53resourcechange
+
+  name    = "staging.api.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  records = ["crossfeed-stage-1792947306.us-gov-west-1.elb.amazonaws.com"]
+  ttl     = 300
+  type    = "CNAME"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+
```

Comment:
These changes should also include an update to the [Resources section of the README](https://github.com/cisagov/cool-dns-cyber.dhs.gov/blob/develop/README.md#resources).  If you have `terraform-docs` installed, you can simply run this command (from the root directory of this project) and commit the relevant changes:
```
terraform-docs markdown --anchor=false --html=false --lockfile=false .
```

Reply:
@dav3r These changes have been made.